cmd/ocagent/config.go

```diff
@@ -20,26 +20,26 @@ import (
 
 const defaultOCInterceptorAddress = "localhost:55678"
 
-type topLevelConfig struct {
-	OpenCensusInterceptorConfig *config `yaml:"opencensus_interceptor"`
+type config struct {
+	OpenCensusInterceptorConfig *interceptorConfig `yaml:"opencensus_interceptor"`
 }
 
-type config struct {
+type interceptorConfig struct {
 	// The address to which the OpenCensus interceptor will be bound and run on.
 	Address string `yaml:"address"`
 }
 
-func (tcfg *topLevelConfig) openCensusInterceptorAddressOrDefault() string {
-	if tcfg == nil || tcfg.OpenCensusInterceptorConfig == nil || tcfg.OpenCensusInterceptorConfig.Address == "" {
+func (c *config) interceptorAddress() string {
```

Comment:
Please revert this, we are going to have more than just the OpenCensus interceptor:
* OpenCensus interceptor
* Zipkin interceptor
* Istio Mixer
* Jaeger
and many more

Please take a look at https://github.com/census-instrumentation/opencensus-service#implementation-details-of-agent-server